### DISCUSSION TITLE
Welcome to ClinicoPathJamoviModule Discussions!

### DISCUSSION THREAD

**Discussion by sbalci** [sbalci/ClinicoPathJamoviModule#37] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ilias-nikolakopoulos** [comment]
Hello! Thanks for making the module. Quick question: The pairwise comparisons for survival analysis are using the Bonferroni adjustment for p-values, or something else?
Thanks!

**Reply by sbalci** [reply]
Hi,
It uses Benjamini & Hochberg as described here: https://stat.ethz.ch/R-manual/R-devel/library/stats/html/p.adjust.html

I just used the default, but I can make it optional as well. Do you have any preference or suggestion?

**Reply by ilias-nikolakopoulos** [reply]
Thanks for the quick reply! No preference, it would be nice to be able to select a specific correction (Bonferroni, Holm, etc).
Also, I would like to ask if there can be an option to make an adjusted survival plot with more than one variable. 

Thanks again for the terrific work!

**Reply by sbalci** [reply]
Thank you. 
I have taken notes as issues. I will work on them.

**Reply by ilias-nikolakopoulos** [reply]
Amazing! Thank you so much!!

**Comment by sinarafaelpotel** [comment]
Hello and thank you very much for the terrific work on your module!

Unfortunately, I am having trouble installing the latest ClinicoPath Descriptive version (ClinicoPathDescriptives-0.0.2.0039.jmo) on Jamovi (my version : 2.4, windows). It says that the module is not compatible when I add it manually, and it does not appear on the library modules list. The only available modules are jsurvival (which works), jjstatsplots and meddecide (not installed). 

Do you have any idea why it happens? 

Best regards, 

Sina

**Reply by sbalci** [reply]
Hi,
Did you try to install within jamovi?
That version is not working and had update. You may download the current module here: https://library.jamovi.org/win64/R4.1.3/ClinicoPathDescriptives-0.0.2.05.jmo and install it via side load. as here: https://github.com/sbalci/ClinicoPathJamoviModule/raw/master/man/figures/jamovi-sideload.gif

**Reply by sinarafaelpotel** [reply]
Hi Serdar, 

It works perfectly, many thanks!
